README.md

```diff
@@ -1,10 +1,20 @@
-[![Build Status](https://travis-ci.org/dedis/cosi.svg?branch=master)](https://travis-ci.org/dedis/cosi)
-[![Coverage Status](https://coveralls.io/repos/github/dedis/cosi/badge.svg?branch=master)](https://coveralls.io/github/dedis/cosi?branch=master)
+[![Build Status](https://travis-ci.org/dedis/cothority.svg?branch=master)](https://travis-ci.org/dedis/cothority)
+[![Coverage Status](https://coveralls.io/repos/github/dedis/cothority/badge.svg?branch=master)](https://coveralls.io/github/dedis/costhority?branch=master)
 
 
 # CoSi
 
-This repository implements the CoSi protocol for scalable collective signing.
+**Update: Development of CoSi moved to the 
+[cothority](https://github.com/dedis/cothority/) repository. 
+For an updated version of this document see: 
+https://github.com/dedis/cothority/blob/master/app/cosi
+This repository only contains the first unstable 
+[`v0` branch](https://gopkg.in/dedis/cosi.v0) and some outdated 
```

Comment:
This still looks too 'alive' for me. I thought of really just having the first two bold paragraphs, then a third paragraph with a short explanation and a link to the v0-branch.

Or if you think this is too much cutting, I'd change the sentence in 'Versions' to:

For the moment we have two version: v0 [link] and the master-branch of the cothority-repository, where development continues [link].